.papr.yml

```diff
@@ -0,0 +1,34 @@
+branches:
+  - master
+
+host:
+  distro: fedora/26/cloud
```

Comment:
Why not `fedora/26/atomic` here? Isn't that more aligned with what this tool is targeted at?

For an example of how this can work, see the pattern in projectatomic/atomic, where we just build and e.g. pylint in a container, but then install onto the host and run the integration tests natively. (And just like projectatomic/atomic, a second `fedora/26/cloud` testsuite can always be added).

Reply:
@jlebon because IRC 😀